projects/ziti-console-lib/package.json

```diff
@@ -22,7 +22,7 @@
     "lottie-web": "^5.12.2",
     "ngx-lottie": "^10.0.0",
     "primeng": "^16.0.2",
-    "vanilla-jsoneditor": "^0.18.2",
+    "vanilla-jsoneditor": "^0.23.4",
```

Comment:
changed to match parent project's version of same dep

Reply:
Actually I think we're up to version `^1.1.2` of the vanilla-jsoneditor, via `package.json` in the `ziti-console` root

Reply:
Good catch. Fixed.